config.json

```diff
@@ -1,7 +1,7 @@
 {
   "embedding": {
     "provider": "openai",
-    "model": "text-embedding-3-large",
+    "model": "text-embedding-3-small",
```

Comment:
`text-embedding-3-large` has a dimension size of 3072. Changing the dimension to match this did not work, but moving to `text-embedding-3-small` did work.